btree/btree.c

```diff
@@ -2866,8 +2978,16 @@ static bool ff_isfit(struct slot *slot)
 	if (h->ff_level == 0 &&
 	   ff_crctype_get(slot->s_node) == M0_BCT_BTREE_ENC_RAW_HASH)
 		crc_size = CRC_VALUE_SIZE;
-	return h->ff_ksize + ff_valsize(slot->s_node) + crc_size <=
-	       ff_space(slot->s_node);
+	if (IS_EMBEDDED_INDIRECT(slot->s_node)) {
```

Comment:
The whole function ff_isfit can be defined as
static bool ff_isfit(struct slot *slot)
{
 return !ff_isoverflow();
}

Reply:
Done.